include/tvm/runtime/ndarray.h

```diff
@@ -257,10 +257,10 @@ class NDArray::Container {
    *  Default value 0 means normal NDArray::Conatainer.
    *
    *  We can extend a more specialized NDArray::Container
-   *  and use the array_type_index_ to indicate
+   *  and use the array_type_info_ to indicate
    *  the specific array subclass.
    */
-  int32_t array_type_index_{0};
+  int32_t array_type_info_{0};
```

Comment:
consider name the specific field as array_type_code_,  it is still consistent with array_type_info::code